byron/ledger/formal-spec/utxo.tex

```diff
@@ -209,22 +209,21 @@ \subsection{Witnesses}
     \begin{array}{r@{~\in~}lr}
       \fun{wits} & \Tx \to \powerset{(\VKey \times \Sig)}
       & \text{witnesses of a transaction}\\
-      \fun{hash_{spend}} & \Addr \mapsto \HashKey
-      & \text{hash of a spending key in an address}\\
+      \fun{hash_{vkey}} & \Addr \mapsto \HashKey
```

Comment:
I'm fine with either names, but I think we changed this to make it consistent with the Shelley spec. @JaredCorduan ?

Reply:
In the Shelley spec we need to make the distinction between the payment keys (signing and verifying) and the stake keys (public and verifying). The word "pay" instead of "spend" would make this consistent with Shelley. I was talking with @Jimbo4350 earlier, and he thought that the "spending" key sounded like it would be a private key, which is understandable. I think "pay" would have this same flaw. So I suggested "verification" for Byron to stress that it is the "public" key.  But I had forgotten about consistency with the Shelley spec, which we probably want. Perhaps we can just stress this in the prose?

Reply:
Sure. Makes sense. @Jimbo4350 ?

Reply:
Just to be clear, revert this change and clarify the meaning of spending key in the prose?
@dnadales